### PR TITLE
Increase device pixel ratio for printing

### PR DESCRIPTION
**Pull request by SebCorbin**

### CODE REVIEW

File: src/modules/Map/Map/components/PrintControl/export.js [hunk 1]
```diff
@@ -32,10 +38,18 @@ export default async function exportPdf (map, orientation, format = 'a4') {
     ignoreElements: ({ className: classes }) => typeof classes === 'string' && classes.includes('mapboxgl-control-container'),
   });
 
-  // JsPdf will automatically detect and put the image full page
-  doc.addImage(renderedContainer, 'PNG', 0, 0);
+  // Put the image full page
+  const conversionFactor = 96 / 25.4;
```
Comment (dtrucs) [inline src/modules/Map/Map/components/PrintControl/export.js]: What do these numbers mean?

Comment (SebCorbin) [reply src/modules/Map/Map/components/PrintControl/export.js]: From pixels to mm

Comment (dtrucs) [reply src/modules/Map/Map/components/PrintControl/export.js]: Sorry for the incomprehension, it was a suggestion to define these numbers with explicit const.

But after another review, i wonder where the `dpi / 96` computation is optimized for print? Is it a rule or have we passed a resolution high enough to say that everything is fine?

Comment (jrmi) [reply src/modules/Map/Map/components/PrintControl/export.js]: I suggest that you add a comment to explain the meaning of this values and don't go further for now as this feature need a longer thinking, if you see what I mean. It's ok for you @dtrucs  ?

**Pull request merged**